cfgmgr/teammgr.cpp

```diff
@@ -654,10 +656,46 @@ bool TeamMgr::removeLag(const string &alias)
 
     stringstream cmd;
     string res;
+    pid_t pid;
 
     cmd << TEAMD_CMD << " -k -t " << shellquote(alias);
     EXEC_WITH_ERROR_THROW(cmd.str(), res);
 
+    try
+    {
+        std::stringstream cmd;
+        cmd << "cat " << shellquote("/var/run/teamd/" + alias + ".pid");
+        EXEC_WITH_ERROR_THROW(cmd.str(), res);
+    }
+    catch (const std::exception &e)
+    {
+        SWSS_LOG_NOTICE("Failed to remove non-existent port channel %s pid...", alias.c_str());
+        return false;
+    }
```

Comment:
Thinking about this a bit more, IMO this is internal implementation details of `teamd`, it's better not assume the location and naming of the file.

Reply:
Teammgrd already reads this file in `cleanTeamProcesses`, and I don't think there's another easy way to get the teamd PID.

Reply:
`cleanTeamProcesses `has most part of the code. Is it good to reuse the same code by passing pid? if no pid passed, then original flow. Just a thought